README.md

```diff
@@ -401,7 +409,123 @@ via [additional environment variables](https://github.com/FIWARE/helm-charts/blo
 
 ### IAM components
 
-> (similar to the Access Node section)
+The [DOME IAM-Framework](https://github.com/DOME-Marketplace/iam-components) is a set of microservices, that enables
+users in the DOME ecosystem to authenticate into the [DOME Marketplace](https://dome-marketplace.org).
```

Comment:
Maybe we could add a sentence, that the authentication process itself is further described below in the #authentication section